file_upload/migrations/0002_alter_document_location.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.9 on 2021-12-04 13:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('file_upload', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='document',
+            name='location',
+            field=models.FileField(upload_to='file_upload_assets/'),
+        ),
+    ]
```

Comment:
Is it necessary to make 2 different migrations here? if not, please remove the first migration and make one.